chain/indexer.go

```diff
@@ -432,7 +432,7 @@ func (t *TipSetIndexer) runActorProcessor(ctx context.Context, p ActorProcessor,
 	}
 }
 
-func (t *TipSetIndexer) Close() error {
+func (t *TipSetIndexer) closeProcessors() error {
```

Comment:
Commentary: Should we be returning the errors in addition to logging them, or should we drop the return param from both of these close methods?

Reply:
I think in this case we want to log the errors because we want to continue with all the close calls, not exit at first error.

We could remove the error return. I tend to keep them assuming that one day we will return an error and the caller should handle it. But that's not a very strong argument to keep it. What do you think?

Reply:
I'd be in favor of nixing the errors, it makes the code a bit more readable -- but these aren't strong feelings. I'll leave it up to you.